system/settings/settings.xml

```diff
@@ -49,6 +49,9 @@
         <setting id="videoplayer.usedisplayasclock" type="boolean" label="13510" help="36166">
           <level>1</level>
           <default>false</default>
+          <dependencies>
+            <dependency type="enable" setting="audiooutput.passthrough" operator="is">false</dependency>
```

Comment:
Take great care. Check the following:
 User enables Sync Playback to Display! After that he enables PT, now - the Sync Playback to Display Setting disappears - but is still ticked and enabled.